scilpy/tractograms/tractogram_operations.py

```diff
@@ -201,8 +202,7 @@ def perform_tractogram_operation_on_sft(op_name, sft_list, precision,
         The final SFT
     """
     # Performing operation
```

Comment:
I know you didn't change this line but that comment isn't particularly useful no?